master/buildbot/status/words.py

```diff
@@ -822,7 +822,7 @@ def usageError(f):
             def logErr(f):
                 klog.err_json(f)
                 self.send("Something bad happened (see logs)")
-            d.addErrback(log.err)
+            d.addErrback(klog.err_json)
```

Comment:
I think you can remove this errorback. As you can see above, there is klog decorated by `addErrback`